### DISCUSSION TITLE
Autogen Keep Using OpenAI API, Not Local

### DISCUSSION THREAD

**Discussion by Kare-Udon** [microsoft/autogen#746] [Q&A] [answered]
I have followed the tutorial of https://microsoft.github.io/autogen/docs/reference/oai/completion/, start the FastAI servers.

My code is

```
from autogen import oai

response = oai.ChatCompletion.create(
    config_list=[
        {
            "model": "chatglm3-6b",
            "base_url": "http://localhost:8000/v1",
            "api_type": "open_ai",
            "api_key": "NULL", # just a placeholder
        }
    ],
    messages=[{"role": "user", "content": "Hi"}]
)

print(response)
```

And Autogen keeps requesting OpenAI API instead of localhost.

`openai.error.Timeout: Request timed out: HTTPSConnectionPool(host='api.openai.com', port=443): Max retries exceeded with url: /v1/chat/completions`

My Autogen version is `0.1.14`

What is going wrong? Could someone help me please?

**Comment by rickyloynd-microsoft** [comment, marked as answer]
`pyautogen==0.1.14` uses the old version of the openai package, `openai==0.28.1`. But most parts of the repo are for the newer version, `pyautogen==0.2.0b6`, and `openai==1.x`. `base_url` is for the newer versions. You can replace `base_url` with the old `api_base`. Hopefully that helps.

**Reply by Kare-Udon** [reply]
That is the right solution! Thank you for your warmly assist.
